### ISSUE TITLE
2.2.8 - Sender Name

### ISSUE THREAD

**Issue by trucatchtraps** [sashas777/bug-from-email#17]
Hi, I just installed Magento 2.2.8 and there is a small issue with this extension. When the confirmation emails are being sent, they are not using the "Sender Name" but instead using the "Sender Email". Please advise a fix please. Thanks!

**Comment by sashas777** [comment]
I not sure what causes it. So far some people confirmed it worked. May be there something else what cuase this issue? Do you have Sender Name set at the admin?

**Comment by trucatchtraps** [comment]
Yes Sender name is set. Was working with 2.2.7 but when I upgraded to 2.2.8 started using the email address instead.

**Comment by xroygbiv** [comment]
@trucatchtraps I have the same issue on 2.2.8 
(also, the problem can be reproduced on a clean Magento installation)

here's what I found: 
Zend\Mail\Message expects two arguments
/vendor/zendframework/zend-mail/src/Message.php:144 
```php
   public function setFrom($emailOrAddressList, $name = null)
    {
        $this->clearHeaderByName('from');
        return $this->addFrom($emailOrAddressList, $name);
    }
```
however, the second argument is missing in this method: 
/vendor/magento/framework/Mail/Message.php:93
```php
    public function setFrom($fromAddress)
    {
        $this->zendMessage->setFrom($fromAddress);
        return $this;
    }
```

You can override the method just by adding a second argument
```php
    public function setFrom($fromAddress, $name = null)
    {
        $this->zendMessage->setFrom($fromAddress, $name);
        return $this;
    }
```
